README.md

```diff
@@ -1,8 +1,10 @@
 Peer Book JavaScript Implementation
 ===================================
 
-[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://ipn.io)
-[![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23ipfs)
+[![](https://img.shields.io/badge/made%20by-Protocol%20Labs-blue.svg?style=flat-square)](http://protocol.ai)
+[![](https://img.shields.io/badge/project-libp2p-yellow.svg?style=flat-square)](http://libp2p.io/)
+[![](https://img.shields.io/badge/freenode-%23libp2p-yellow.svg?style=flat-square)](http://webchat.freenode.net/?channels=%23libp2p)
+[![Discourse posts](https://img.shields.io/discourse/https/discuss.libp2p.io/posts.svg)](https://discuss.libp2p.io)
 [![Build Status](https://travis-ci.org/libp2p/js-peer-book.svg?style=flat-square)](https://travis-ci.org/libp2p/js-peer-book)
 [![Coverage Status](https://coveralls.io/repos/github/libp2p/js-peer-book/badge.svg?branch=master)](https://coveralls.io/github/libp2p/js-peer-book?branch=master)
```

Comment:
Need to update travis and coveralls. I updated the badge generator gist I use to have discourse. https://gist.github.com/jacobheun/1da73586478b6e0de976119bd250392b in case you want to leverage that. It also uses the same flat-square badge format as the other badges to keep things matching